src/backend/app.js

```diff
@@ -21,10 +21,15 @@ app.use(cors());
 
```

Comment:
In this file you should import the meals and the reviews router, and write the following lines of code, to map these routers to their appropriate paths:
```javascript
const mealsRouter = require("./api/meals");
const reviewsRouter = require("./api/reviews");

// other code

router.use("/meals", mealsRouter);
router.use("/reviews", reviewsRouter);
```